### ISSUE TITLE
Max width for text blocks should be 45em

### ISSUE THREAD

**Issue by jvine** [sul-dlss/SearchWorks#640]
http://searchworks-test.stanford.edu/redesign/view/fp045wx3019

The maxwidth for dd has been set globally, but this record has an abstract that's not in a dd.

BTW the html is a bit odd with a div in a dl...

```
      <dl>
            <div class="section-abstract">
      Congeners of the sea anemone...
 </div>
      </dl>
```

But bottom line -  .section-abstract should have max-width of 45em.
